models/perm/access/repo_permission.go

```diff
@@ -336,53 +333,46 @@ func isUserRepoAdmin(ctx context.Context, repo *repo_model.Repository, user *use
 
 // AccessLevel returns the Access a user has to a repository. Will return NoneAccess if the
 // user does not have access.
-func AccessLevel(user *user_model.User, repo *repo_model.Repository) (perm_model.AccessMode, error) {
+func AccessLevel(user *user_model.User, repo *repo_model.Repository) (perm_model.AccessMode, error) { //nolint
```

Comment:
Why nolint?

Reply:
Because linter ask the function name to be changed to `Level(`, but I think `AccessLevel` is more meaningful.